### PR TITLE
feat: allow setting of reverse DNS records

### PR DESCRIPTION
**Pull request by morremeyer**
This allows setting reverse DNS records for the Load Balancer with the annotations:

* `load-balancer.hetzner.cloud/ipv4-rdns` for IPv4 and
* `load-balancer.hetzner.cloud/ipv6-rdns` for IPv6.

As this is my first time working with testify and contributing to this repo, please check the tests I wrote carefully.

They pass, but some feedback if I wrote them correctly would be much appreciated, especially regarding the repeated type conversions in the mocks.

### CODE REVIEW

**Comment by morremeyer** [comment]
This is definitely not stale, waiting for maintainer review.

**Review by fhofherr — APPROVED** [review]

**Review by LKaemmerling — APPROVED** [review]

**Comment by LKaemmerling** [comment]
Hey @morremeyer,

sorry for the late reply, we were actually quite busy the last couple of months. 

Good job! We will merge it soon!

**Comment by morremeyer** [comment]
Moin @LKaemmerling, is there a plan for when to release the current state of `main` as a new version?

**Pull request merged**
